libs/api/libbpf_map.cpp

```diff
@@ -392,9 +408,10 @@ ring_buffer__new(int map_fd, ring_buffer_sample_fn sample_cb, void* ctx, const s
 void
 ring_buffer__free(struct ring_buffer* ring_buffer)
 {
-    for (auto it = ring_buffer->subscriptions.begin(); it != ring_buffer->subscriptions.end(); it++) {
-        (void)ebpf_ring_buffer_map_unsubscribe(*it);
+    for (auto& sub : ring_buffer->subscriptions) {
+        ebpf_map_unsubscribe(sub);
```

Comment:
```suggestion
    for (auto& subscription : ring_buffer->subscriptions) {
        ebpf_map_unsubscribe(subscription);
```
[Coding conventions](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#coding-conventions) says
> **DON'T** use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be num_widgets instead of widget_count, and opt_widgets instead of option_widgets or optional_widgets.